### PR TITLE
Print styles

### PR DESCRIPTION
**Pull request by breakdancingcat**
Print styles that were noticed when printing law pages
Menu top, breadcrumbs
Screen-Shot-2019-11-01-at-11 17 20-AM
Screen-Shot-2019-11-01-at-11 17 28-AM

### CODE REVIEW

File: resources/scss/components/_content.scss [hunk 1]
```diff
@@ -1,8 +1,3 @@
-// Contact footer content
-footer .content a {
```
Comment (breakdancingcat) [inline resources/scss/components/_content.scss]: This specificity was getting in the way. Luckily I had a solution implemented already (.white-links in global.scss) that I used for the contact footer and removed this code.

**Comment by nickdenardis** [comment]
Is there an after screenshot?

**Comment by breakdancingcat** [comment]
> Is there an after screenshot?

@nickdenardis The screen shots have been in the description since I submitted it.

**Comment by nickdenardis** [comment]
Would love to see a before and after of this now with the newest changes.

**Comment by breakdancingcat** [comment]
@nickdenardis How are you not seeing my screenshots? lol They've been in here!!

**Comment by nickdenardis** [comment]
The edited PR description with before and after shots look great! 👍

**Pull request closed without merging**